### ISSUE TITLE
RPM package for Mageia 8 is broken

### ISSUE THREAD

**Issue by pkreuzt** [polychromatic/polychromatic#462]
After adding OBS repo, trying to install with dnf gives the following error due to unresolved dependencies:

```
Error: 
 Problem: conflicting requests
  - nothing provides typelib(AppIndicator3) needed by polychromatic-0.8.0-1.1.noarch
  - nothing provides libqt5-qtwebengine needed by polychromatic-0.8.0-1.1.noarch
  - nothing provides qt5-qtsvg needed by polychromatic-0.8.0-1.1.noarch
  - nothing provides typelib(AppIndicator3) = 0.1 needed by polychromatic-0.8.0-1.1.noarch
```

Not sure about typelib, but the other "missed" dependencies seem due to different naming of packages. On this system I have installed qt5svg and qtwebengine5 packages. 

For that typelib dependency I have lib64ayatana-appindicator3_1, could this one work?

**Comment by lah7** [comment]
Sorry about that. Looks like the package names changed (and you could be the first Mageia 8 user here!)

I think these would be right:

* `typelib(AppIndicator3)`  → `lib64ayatana-appindicator3_1`
* `libqt5-qtwebengine` → `python3-qt5-webengine`  (but could be `qtwebengine5`)
* `qt5-qtsvg`  → `qtsvg5`

I'll contact @z3ntu to seek help with this. There is a single [`polychromatic.spec`](https://build.opensuse.org/package/view_file/hardware:razer/polychromatic/polychromatic.spec?expand=1) that builds for multiple RPM-based distros.

Not being familiar with RPM myself, I'm not sure it's possible to "ignore" the dependency issues in the meantime.

**Comment by pkreuzt** [comment]
Actually running from git with those packages installed works. Since the Qt environment and naming is confusing (for me at least) I cannot tell which are the good ones. But yesterday, with only the packages I named, it already launched. Didn't test it thoroughly.

There's a way to install packages with rpm without matching deps, but it's not really advisable since binaries won't be integrated in the system and probably won't work either.

**Comment by lah7** [comment]
Good to know! At least in theory, `python3-qt5-webengine` should depend on `qtwebengine5`, which is needed for the effect editor.

When I was checking from a Mageia 8 live CD, I did notice some of these packages were already installed too.

We should get the package sorted this weekend, hopefully.

**Comment by lah7** [comment]
A new package has been released: `0.8.0-2` (Thanks @z3ntu!)
